drivers/virtualbox/virtualbox.go

```diff
@@ -136,7 +136,7 @@ func GetCreateFlags() []cli.Flag {
 }
 
 func (d *Driver) GetSSHHostname() (string, error) {
-	return "localhost", nil
+	return "127.0.0.1", nil
```

Comment:
This is a tradeoff based on the fact that many users on Windows were getting `::1` (ipv6) for `localhost`.  We need to ensure IPv4.